doc/api/async_hooks.md

```diff
@@ -329,20 +329,17 @@ async_hooks.createHook({
   },
   before(asyncId) {
     const indentStr = ' '.repeat(indent);
-    fs.writeFileSync('log.out',
-                     `${indentStr}before:  ${asyncId}\n`, { flag: 'a' });
+    fs.writeSync(1, `${indentStr}before:  ${asyncId}\n`);
```

Comment:
This is going to be pretty subtle for most general Node.js users, I'm afraid. If we're going to go with this, a comment that `fs.writeSync(1...`) means that we're writing to stdout would be helpful.

Reply:
Or `process.stdout.fd`

Reply:
```suggestion
    fs.writeSync(process.stdout.fd, `${indentStr}before:  ${asyncId}\n`);
```

Reply:
`fs.wrteSync(1, ...)` is already used at several places in this file so it's a good step to get it consistent. But using `process.stdout.fd` seems to be even better.

Reply:
Agree, initially I just streamlined the example because part of code was writing to `log.out` and part to the output therefore you could never get the mentioned output by running the example yourself.